packages/roosterjs-editor-core/lib/corePlugins/utils/InlineEntityHandlers/isDelimiter.ts

```diff
@@ -0,0 +1,15 @@
+import { DELIMITER_AFTER, DELIMITER_BEFORE, DelimiterType } from './constants';
+
+export function isDelimiter(element: Element | null | undefined): DelimiterType | null {
+    if (!element) {
+        return null;
+    }
+
+    if (element.id.indexOf(DELIMITER_AFTER) > -1) {
```

Comment:
Instead of using ID, I prefer using className, so that no need to make it unique. And to find a delimiter of a given entity,  we just need to use previousSibling and nextSibling of entity element, and check its class.

Reply:
Sounds good. started using classname